.github/workflows/detect_api_changes.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     - name: Select latest Xcode
       uses: maxim-lobanov/setup-xcode@v1
       with:
-        xcode-version: '16.1'
+        xcode-version: '16.2'
```

Comment:
Should we keep Xcode version the same across all jobs? This one is different from the rest